themes/coderdojoschoeneweide/layouts/_default/baseof.html

```diff
@@ -21,19 +21,41 @@
 
   <body>
     {{ block "header.html" . }}
-    <header class="site-header wrapper">
-      <label id="switch" class="switch">
-        <span style="visibility: hidden;">theme</span>
-        <input type="checkbox" id="slider" title="theme" />
-        <span class="slider round"></span>
-      </label>
-      <a class="site-title site-title-light" href="/">
-        <img
-          class="site-logo"
-          src="/images/logo.webp"
-          alt="CoderDojo Schoeneweide Logo"
-        />
-      </a>
+    <header class="site-header wrapper" role="banner">
+        <div class="abod-lang">
+            {{ range .Translations }}
+              {{ if eq .Lang "en" }}
+                  <li class="hover:bg-slate-300 p-2">
+                    <a href="{{ .Permalink }}">
+                        <img src="/images/flags/usa.png" alt="  ">
+                        {{ "  " | .RenderString }} English</a>
+                  </li>
+              {{ else if eq .Lang "de" }}
+                  <li class="hover:bg-slate-300 p-2">
+                    <a href="{{ .Permalink }}">
+                        <img src="/images/flags/deutschland.png" alt=" ">
+                        {{ "  " | .RenderString }} Deutsch</a>
+                  </li>
+              {{ else if eq .Lang "ar" }}
+              <li class="hover:bg-slate-300 p-2">
+              <a href="{{ .Permalink }}">
+                <img src="/images/flags/jemen.png" alt=" ">
+                {{ "  " | .RenderString }} العربية</a>
+              </li>
+              {{ end }}
+            {{ end }}
+          </div>
+        <label id="switch" class="switch">
+            <input type="checkbox" id="slider"/>
+            <span class="slider round"></span>
+        </label>
```

Comment:
Könntest Du hier noch ein bisschen mit CSS nachbessern, damit es etwas schöner aussieht? Also sowas in der Art wie mein Vorschlag hier?

```suggestion
        <div style="display: flex; justify-content: space-between">
            <ul style="list-style: none; padding-left: 0; display:flex; gap: 1em;">
                {{ range .Translations }}
                {{ if eq .Lang "en" }}
                <li>
                    <a href="{{ .Permalink }}" style="display: flex; gap: 0.5em; align-items: center">
                        <img src="/images/flags/usa.png" alt="  ">
                        {{ " " | .RenderString }} English</a>
                </li>
                {{ else if eq .Lang "de" }}
                <li>
                    <a href="{{ .Permalink }}" style="display: flex; gap: 0.5em; align-items: center">
                        <img src="/images/flags/deutschland.png" alt=" ">
                        {{ " " | .RenderString }} Deutsch</a>
                </li>
                {{ else if eq .Lang "ar" }}
                <li>
                    <a href="{{ .Permalink }}" style="display: flex; gap: 0.5em; align-items: center">
                        <img src="/images/flags/jemen.png" alt=" ">
                        {{ " " | .RenderString }} العربية</a>
                </li>
                {{ end }}
                {{ end }}
            </ul>
            <label id="switch" class="switch">
                <input type="checkbox" id="slider"/>
                <span class="slider round"></span>
            </label>
        </div>
```